### PR TITLE
Finish m6 implementation

### LINKED ISSUE #313: Benchmark script/tx fees for common cases
I think that could be done by just extracting fees from E2E test case. Like with some instrumentation (structured logs?) for that case.

### LINKED ISSUE #323: Delegate-to-Platform-server interaction

### LINKED ISSUE #328: CLI support for bidder approval using Platform server
- [ ] Publish bidder approval on Platform server (new CLI action for seller?)
- [ ] Query for bidder approval and use it in `newBid`

### PR DESCRIPTION
**Pull request by uhbif19**
Closes #328
Closes #323 
Closes #313

### CODE REVIEW

File: src-utils/HydraAuctionUtils/Hydra/Runner.hs [hunk 1]
```diff
@@ -160,24 +165,43 @@ executeHydraRunner ::
...
 executeHydraRunnerFakingParams ::
-  HydraClient -> HydraRunner a -> WithActorT L1Runner a
+  HasCallStack =>
+  forall a.
+  RealProtocolClient HydraProtocol ->
+  HydraRunner a ->
+  WithActorT L1Runner a
 executeHydraRunnerFakingParams node monad = do
+  (_, !scriptRegistry) <- lift prepareScriptRegistry
+  liftIO $ putStrLn "AFTER PREPARE"
```
Comment (nazrhom) [inline src-utils/HydraAuctionUtils/Hydra/Runner.hs]:
```suggestion
```

Comment (uhbif19) [reply src-utils/HydraAuctionUtils/Hydra/Runner.hs]: This is from WIP commit.

File: src-utils/HydraAuctionUtils/Tx/AutoCreateTx.hs [hunk 2]
```diff
@@ -258,5 +258,6 @@ autoSubmitAndAwaitTx ::
   m Tx
 autoSubmitAndAwaitTx params = do
   tx <- autoCreateTx params
+  -- FIXME
```
Comment (nazrhom) [inline src-utils/HydraAuctionUtils/Tx/AutoCreateTx.hs:264]: What is this for?

Comment (uhbif19) [reply src-utils/HydraAuctionUtils/Tx/AutoCreateTx.hs:264]: It ignores Left, which might be used.

https://github.com/mlabs-haskell/hydra-auction/issues/277

**Review by nazrhom — APPROVED** [review]

**Pull request merged**